CMakeLists.txt

```diff
@@ -73,10 +73,14 @@ add_executable(astra_camera_node
 target_link_libraries(astra_camera_node astra_driver_lib)
 add_dependencies(astra_camera_node astra_openni2)
 
-install(TARGETS astra_camera_node astra_wrapper astra_driver_lib
+install(TARGETS astra_camera_node
   DESTINATION ${CMAKE_INSTALL_PREFIX}/bin/
 )
 
+install(TARGETS astra_wrapper astra_driver_lib
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/
```

Comment:
I would recommend writing platform independent code here even if currently the focus is on Linux only, e.g. https://github.com/ament/ament_index/blob/f5421672e5dc841afc52ef380ded5bf40f682a32/ament_index_cpp/CMakeLists.txt#L45-L47